src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -269,7 +274,12 @@ public static Translation2d getWristTipPosition(double armPos, double wristPos)
 
     public boolean wristMovementForbidden(double armPos, double wristPos, double wristVelSign) {
         // If the position is not forbidden, then the movement is not forbidden
-        if(!positionForbidden(armPos, wristPos)) return false;
+        if(positionForbidden(armPos, wristPos)) {
+            forbFlag = true;
+            return false;
+        }
```

Comment:
Why did you change this portion of the code? Do you think this is the place in the code where it detects that wrist movement is forbidden?

Reply:
Hint: `wristMovementForbidden` does tell you whether wrist movement is forbidden (like it says in the method), but does it detect whether the manipulator's controls are valid?